stl/inc/format

```diff
@@ -2712,7 +2762,7 @@ template <class _Ty, class _CharT, _Basic_format_arg_type _ArgType>
 struct _Formatter_base {
     using _Pc = basic_format_parse_context<_CharT>;
 
-    typename _Pc::iterator parse(_Pc& _ParseCtx) {
+    constexpr typename _Pc::iterator parse(_Pc& _ParseCtx) {

```

Comment:
Preexisting: Couldnt we get away with 
```suggestion
    constexpr auto parse(_Pc& _ParseCtx) {
```

Reply:
maybe, but the return type has to be _Pc::iterator according to the standard, so this prevents a potential error later on down the line.

It also makes it easier to remember what the function returns :D